crates/runtime/benches/bench.rs

```diff
@@ -284,9 +284,34 @@ async fn run_query_and_record_result(
             max_iter_duration_ms = iter_duration_ms;
         }
 
-        if let Err(e) = res {
-            query_err = Some(e);
-            break;
+        match res {
+            Ok(records) => {
+                let num_rows = records
+                    .iter()
+                    .map(arrow::array::RecordBatch::num_rows)
+                    .sum::<usize>();
+
+                // Show the first 30 records of the result
+                let limited_records = records
+                    .iter()
+                    .take(1)
+                    .map(|x| x.slice(0, x.num_rows().min(30)))
```

Comment:
```suggestion
                    .map(|x| x.slice(0, x.num_rows().min(10)))
```

10 is probably enough.

Reply:
Updated. thank you